e2e/move2kube.sh

```diff
@@ -42,8 +42,6 @@ GIT_ORG="gfarache31/m2k-test"
 GIT_REPO="bitbucket.org/${GIT_ORG}"
 GIT_SOURCE_BRANCH="master"
 GIT_TARGET_BRANCH="e2e-test-$(date +%s)"
-NOTIFICATIONS_DISABLED=true
```

Comment:
I am fine with removing that but it is now less explicit that no notification will be sent and hence that we are not testing that part here (I know, it's already the case for MTA)

Reply:
is there a use case in which the notifications are enabled?

Reply:
well, I think notifications are very important for users so from a ent-to-end standpoint,  testing that notifications are correctly sent and with good message and link would be great

That said, we are never testing them in e2e tests, I think that we are relying on QE to uncover issue with notifications